lib/puma/reactor.rb

```diff
@@ -244,7 +244,8 @@ def run_internal
 
                 cert = ssl_socket.peercert
 
-                c.close
+                # may generate another SSL error, continue to remove the monitor
+                c.close rescue nil
```

Comment:
Was this in response to a specific issue?

Reply:
Yes.  Where, don't recall.  I can comment it out and run the tests.